master/internal/api_command.go

```diff
@@ -144,6 +145,12 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 	config.Resources.ResourcePool = poolName.String()
 	config.Resources.Slots = resources.Slots
 
+	// Apply the scheduler's default priority.
+	if config.Resources.Priority == nil {
+		prio := masterConfig.DefaultPriorityForPool(poolName.String())
+		config.Resources.Priority = &prio
+	}
```

Comment:
```suggestion
	}
	// TODO (CM-493) 
```
^ just to make sure that invariant config overrides occur after this default priority is set and before constraints are checked (the latter keeps consistent with the experiment config policies workflow)